### ISSUE TITLE
Calling Mapbox.getInstance(...) takes around 500ms

### ISSUE THREAD

**Issue by jush** [mapbox/mapbox-gl-native#11171]
**Platform:** Android
**Mapbox SDK version:** 5.4.1

### Steps to trigger behavior

 1. Call `Mapbox.getInstance(...)` with the following code and replace with proper key:
```Kotlin
val startTime = SystemClock.elapsedRealtime()
Mapbox.getInstance(appContext, "pk.....")
val endTime = SystemClock.elapsedRealtime()
Timber.d("It took ${endTime - startTime} to initialize Mapbox")
```
The above code is part of my `Application.onCreate()`  method.

The device I'm using is LG G watch Sport running stock Android 8.0.0 (API 26)

### Expected behavior
Initializing Mapbox should not have such big impact on app start time.

### Actual behavior
Usually it takes 500ms and sometimes it goes even as high as 1000ms

**Comment by tobrun** [comment]
Thank you for the report, looked into the time it took for the different components to execute:

On a Google Pixel it took:
124 ms to invoke Mapbox.getInstance() from which:
 - 91 ms to initialize telemetry
 - 33 to load up the native library

With the upcoming 6.0.0 release we revamped telemetry and its no longer found in `Mapbox#getInstance`. This should improve the loading time significantly. Closing as non-actionable.
